op-chain-ops/state/memory_db.go

```diff
@@ -232,6 +235,9 @@ func (db *MemoryStateDB) SetState(addr common.Address, key, value common.Hash) {
 }
 
 func (db *MemoryStateDB) DeleteState(addr common.Address, key common.Hash) {
+	db.rw.Lock()
+	defer db.rw.Unlock()
+
 	account, ok := db.genesis.Alloc[addr]
 	if !ok {
 		panic(fmt.Sprintf("%s not in state", addr))
```

Comment:
It should be included because it's an interface added by Kroma geth.
[go-ethereum/core/vm/interface.go]
```
// [Scroll: START]
GetRootHash() common.Hash
GetLiveStateAccount(addr common.Address) *types.StateAccount
GetProof(addr common.Address) ([][]byte, error)
GetProofByHash(addrHash common.Hash) ([][]byte, error)
GetStorageProof(a common.Address, key common.Hash) ([][]byte, error)
// [Scroll: END]
```